src/overlays/actors/ovl_En_Wf/z_en_wf.h

```diff
@@ -6,9 +6,86 @@
 
 struct EnWf;
 
+typedef void (*EnWfActionFunc)(struct EnWf*, GlobalContext*);
+
+typedef enum {
+    /*  0 */ WOLFOS_LIMB_NONE,
+    /*  1 */ WOLFOS_LIMB_ROOT,
+    /*  2 */ WOLFOS_LIMB_BACK_LEFT_THIGH,
+    /*  3 */ WOLFOS_LIMB_BACK_LEFT_SHIN,
+    /*  4 */ WOLFOS_LIMB_BACK_LEFT_PASTERN,
+    /*  5 */ WOLFOS_LIMB_BACK_LEFT_PAW,
+    /*  6 */ WOLFOS_LIMB_TAIL,
+    /*  7 */ WOLFOS_LIMB_ABDOMEN,
+    /*  8 */ WOLFOS_LIMB_BACK_RIGHT_THIGH,
+    /*  9 */ WOLFOS_LIMB_BACK_RIGHT_SHIN,
+    /* 10 */ WOLFOS_LIMB_BACK_RIGHT_PASTERN,
+    /* 11 */ WOLFOS_LIMB_BACK_RIGHT_PAW,
+    /* 12 */ WOLFOS_LIMB_THORAX,
+    /* 13 */ WOLFOS_LIMB_FRONT_RIGHT_UPPER_LEG,
+    /* 14 */ WOLFOS_LIMB_FRONT_RIGHT_LOWER_LEG,
+    /* 15 */ WOLFOS_LIMB_FRONT_RIGHT_CLAW,
+    /* 16 */ WOLFOS_LIMB_HEAD_ROOT,
+    /* 17 */ WOLFOS_LIMB_HEAD,
+    /* 18 */ WOLFOS_LIMB_EYES,
+    /* 19 */ WOLFOS_LIMB_FRONT_LEFT_UPPER_LEG,
+    /* 20 */ WOLFOS_LIMB_FRONT_LEFT_LOWER_LEG,
+    /* 21 */ WOLFOS_LIMB_FRONT_LEFT_CLAW,
+    /* 22 */ WOLFOS_LIMB_MAX
+} EnWfLimb;
+
+typedef enum {
+    /*  0 */ WOLFOS_ACTION_WAIT_TO_APPEAR,
+    /*  2 */ WOLFOS_ACTION_DIE = 2,
+    /*  3 */ WOLFOS_ACTION_DAMAGED,
+    /*  4 */ WOLFOS_ACTION_TURN_TOWARDS_PLAYER,
+    /*  5 */ WOLFOS_ACTION_BACKFLIP,
+    /*  6 */ WOLFOS_ACTION_WAIT,
+    /*  7 */ WOLFOS_ACTION_REACT_TO_PLAYER,
+    /*  8 */ WOLFOS_ACTION_FIRST_SLASH,
+    /*  9 */ WOLFOS_ACTION_RUN_AT_PLAYER,
+    /*  10 */ WOLFOS_ACTION_SEARCH_FOR_PLAYER,
+    /*  11 */ WOLFOS_ACTION_RUN_AROUND_PLAYER,
+    /*  12 */ WOLFOS_ACTION_SECOND_SLASH,
+    /*  14 */ WOLFOS_ACTION_SIDESTEP = 14,
+    /*  15 */ WOLFOS_ACTION_STUNNED
+} EnWfAction;
+
 typedef struct EnWf {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x390];
+    /* 0x014C */ Vec3s bodyPartsPos[10];
+    /* 0x0188 */ SkelAnime skelAnime;
+    /* 0x01CC */ Vec3s jointTable[WOLFOS_LIMB_MAX];
+    /* 0x0250 */ Vec3s morphTable[WOLFOS_LIMB_MAX];
+    /* 0x02D4 */ s32 action; // Used in  some sort of actionFunc-checking system
```

Comment:
```suggestion
    /* 0x02D4 */ s32 action; // Used in some sort of actionFunc-checking system
```

Reply:
I think I'll rewrite this anyway now I have a better idea of how it's used.